rbx_dom_lua/src/EncodedValue.lua

```diff
@@ -20,6 +20,41 @@ local function serializeFloat(value)
 	return value
 end
 
+local function enumsEncoder(enum)
```

Comment:
Can you write comments describing why these functions do what they do? Many readers won't be familiar with why bitwise operations would be involved in dealing with axes and faces.

Reply:
Done! Let me know if it needs any adjustments.